forms-flow-api/src/formsflow_api/models/form_process_mapper.py

```diff
@@ -132,6 +133,21 @@ def filter_conditions(cls, **filters):
         query = cls.query.filter(*filter_conditions) if filter_conditions else cls.query
         return query
 
+    @classmethod
+    @user_context
+    def access_filter(cls, query: BaseQuery, **kwargs):
```

Comment:
Whether we can avoid this access_filter function here and reuse tenant_authentication method from models/application.py